cypress/test/layout/Container.cy.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import Container from '../../../components/layout/Container';
+
+describe('Container component', () => {
+  it('renders children correctly', () => {
+    mount(
+      <Container fluid flex wide padding="px-2" className="custom-class">
+        <p>Children content</p>
+      </Container>
+    );
+    cy.get('[data-testid="Container-div"]').should('exist');
+
+    cy.get('[data-testid="Container-div"]').should(
+      'contain.text',
+      'Children content'
+    );
+  });
```

Comment:
Check for other CSS cases as well, when other properties like `fluid`, `wide`, etc. properties are provided.

Reply:
done